### DISCUSSION TITLE
Internal Server Error Occurs Intermittently on Running a New Orchard Core CMS App

### DISCUSSION THREAD

**Discussion by krchome** [OrchardCMS/OrchardCore#9946] [Q&A]
### Describe the bug
While running (Ctrl + F5) the Orchard Core CMS Web application once it is created and requisite NuGet package OrchardCore.Application.Cms.Targets added and finally Startup class is modified, all according to the instructions in the official Orchard Core website documentation at [https://docs.orchardcore.net/en/dev/docs/getting-started/], it comes up with the Internal Server Error (intermittently, not always though) as in the image attached

### To Reproduce

1. Follow the article at https://docs.orchardcore.net/en/dev/docs/getting-started/ (to create a new Orchard Core CMS web application)
2. Run the application from Visual Studio by clicking CTRL + F5
3. The Internal Server Error appears at runtime (see the screenshot below)

### Expected behavior
The Orchard Core Setup dashboard should appear 

![Snag_23ccc69](https://user-images.githubusercontent.com/8474855/126137671-af562608-2073-4772-ad00-1cdd5ef664b9.png)

**Comment by deanmarcussen** [comment]
Much will depend on which version of Orchard Core and ASP.NET Core you are running.

.Net 5.0 removed this method, so if you are running .Net 5.0 you will need to be using the dev feed of Orchard Core https://docs.orchardcore.net/en/dev/docs/getting-started/preview-package-source/

Or use core 3

**Comment by krchome** [comment]
Thanks for the insight and apologies as a forethought I should have written the platform ASP.NET 5 and the NuGet package version latest pre-release 1.0.0-rc2-13450 used .
".Net 5.0 removed this method, so if you are running .Net 5.0 you will need to be using the dev feed of Orchard Core https://docs.orchardcore.net/en/dev/docs/getting-started/preview-package-source/ " Yes, I already followed this article and implemented the steps for using the preview feed. Despite this, I am still getting the runtime error under two different scenarios as below:
Scene 1: As i mentioned in my issue earlier, I got the runtime error on running (without debugging) with CTRL + F5. That is to say it didn't come to the stage of setup 
Scene 2: I deleted the application all together that I created in Scene 1 and created a fresh app and ran the application following the steps in [https://docs.orchardcore.net/en/dev/docs/getting-started/] taking care to check the steps for the preview feed once again. 
Conclusion: This time the application did run and setup page appeared. However on finishing the setup after filling the details, the same stack trace appeared with the runtime error. 
Off course I used ASP.NET 5 in the above scenarios with pre-release version 1.0.0-rc2-13450 taking the steps to use the prevoew feed in both the scenes above. 
With core 3.1 it was working fine as expected that I tried earlier. 
This is an intermittent fault and I should say that the vcery first time I created the app in ASP.NET 5 with the dev feed steps, it worked all right as expected. 
So on three different occassions, three different behavior received. Needs to be checked I guess.

**Comment by deanmarcussen** [comment]
> Off course I used ASP.NET 5 in the above scenarios with pre-release version 1.0.0-rc2-13450

Using ASP.NET 5 will not work with rc2-13450

You must upgrade to the cloudsmith dev feed and version a later release, I suggest the current release of 1.0.0-rc2-16375 or it will NOT work with ASP.NET 5 (I know because I wrote the fix, but we did not backport that to rc2-13450)

**Comment by krchome** [comment]
Hi @deanmarcussen
Thanks very much. 
I didn't find 1.0.0-rc2-16375  release so far. I am using VS 16.10.3 and rc2-13450 seems to be the latest. As I said earlier, I have used cloudsmith dev feed so far with ASP.NET 5 at https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json
Am I missing something?

**Comment by deanmarcussen** [comment]
You are missing something.

Please read the docs here, https://docs.orchardcore.net/en/dev/docs/getting-started/preview-package-source/
and in visual studio check the pre release button.

You can see from the link here what the latest release is https://cloudsmith.io/~orchardcore/repos/preview/packages/detail/nuget/OrchardCore.Application.Cms.Targets/latest/
